ql.go

```diff
@@ -71,7 +71,7 @@ func qlCompile(db *sql.DB) (map[string]*sql.Stmt, error) {
 	stmts := map[string]string{
 		"cleanAddress":  `DELETE FROM Addresses WHERE Seen < now() - duration("2h")`,
 		"cleanRelay":    `DELETE FROM Relays WHERE Seen < now() - duration("2h")`,
-		"cleanDevice":   `DELETE FROM Devices WHERE Seen < now() - duration("24h")`,
+		"cleanDevice":   `DELETE FROM Devices WHERE Seen < now() - duration("168h")`, // 1 week
```

Comment:
Instead of hardcoding a constant, use fmt.Sprintf and the global variable, so that you would only need to change 1 place when we change the value rather than 3.